src/App.jsx

```diff
@@ -4,6 +4,7 @@ import Header from "./components/Header.jsx";
 import HomePage from "./components/HomePage.jsx";
 import ArticleList from "./components/ArticleList.jsx";
 import SingleArticlePage from './components/SingleArticlePage.jsx';
+import TopicList from "./components/TopicList.jsx"
```

Comment:
This line breaks your code since TopicList isn't currently defined. Please remove before continuing!

Reply:
Hi Rose, sorry I had started on the next ticket then realised I hadn't submitted PR and branched so undid some stuff and missed that - I've just removed and pushed to the branch again, is it working now?